view/loadView.js

```diff
@@ -76,17 +76,21 @@ function samlAuthentication(userProfile, done) {
       return Profile.findOrCreate({ where: { name: 'RefocusSSOUser' } });
     }
 
+    // profile already attached - default scope applied on find
     return done(null, user);
   })
   .spread((profile) =>
-    User.create({
+    User.create({ // default scope not applied on create, hence no profile
```

Comment:
Can you check for a user story for this, and if not, please create one?

Reply:
I think I should make it more clear in the comment. Its just a comment, meaning we do not have profile attached here, so we will use User.find to get the attached profile after this (which I did). No further work needed here.